Expensify/expensify-common
# fix: add rules for video conversion

### Fixed Issues
$ https://github.com/Expensify/App/issues/41952 

# Tests

Unit tests added in the PR

# QA

1. Add a comment with both text + video in a chat report
2. Confirm that the video is displayed accordingly
3. Edit comment
4. Expected: The video still shows correctly after editing comment
